templates/miq-template-ext-db.yaml

```diff
@@ -265,7 +265,7 @@ objects:
   spec:
     ports:
       -
-        name: "postgresql"
+        name: "postgres"
```

Comment:
I would rather use the full name "postgresql". Is there a reason you chose to standardize on "postgres"?